test/e2e/create_test.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/openshift/windows-machine-config-operator/test/e2e/clusterinfo"
 )
 
+// vmConfigurationTime is the maximum amount of time expected for a Windows VM to be fully configured and ready for WMCO
+// after the hardware is provisioned.
+const vmConfigurationTime = 8 * time.Minute
```

Comment:
We use [10 minutes](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/retry/retry.go#L11) for SSH connectivity to go through in the WMCO code. Shouldn't we match that? PS: not asking to reuse the same const but to match the time.